modules/node_modules/@ncigdc/components/DownloadButton.js

```diff
@@ -28,14 +31,20 @@ const DownloadButton = ({
   filters = {},
   active,
   setActive,
+  altMessage = false,
+  style = {},
   extraParams = {},
+  dispatch,
 }) => {
-  const icon = <i key="icon" className={`fa fa-download ${active ? 'fa-spinner fa-pulse' : ''}`} />;
+  const icon = active ? <Spinner /> : <DownloadIcon />;
```

Comment:
This works:
```html
<button aria-label="download">
   <div class="fa fa-download"></div>
</button>
```

This also works
```html
<button aria-label="download">
   <div class="fa fa-download"></div>
   <div class="visually-hidden">Download</div>
</button>
```

This does _not_ work (wave shows a red "empty button" error)

```html
<button>
   <div aria-label="download" class="fa fa-download"></div>
</button>
```

The aria-label either needs to be on the button, or the button needs to have text to explain what it does

Quoting the MDN link, "The aria-label attribute is used to define a string that labels the **current element**" (emphasis mine). The "current element" is the one that receives focus when tabbed through the page.

The aria label for the icon would also technically be "download icon", but since people using screenreaders wouldn't care about the existence of an icon whose sole existence is to inform sighted users of the purpose of the button, it'd be preferable to leave that unlabled (if it were an img tag instead of just a blank div, it'd be preferable to set it as aria-hidden)